composer.json

```
@@ -34,12 +34,13 @@
         "twig/twig": "^2.12 || ^3.0"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^2.4",
+        "doctrine/dbal": "^2.13.9",
```

```suggestion
        "doctrine/dbal": "^3.4",
```

We are dropping dbal 2 from all repos.

---

I was checking that if I do that, because of https://github.com/KnpLabs/DoctrineBehaviors/pull/682/files, some symfony dependencies will be raised to 5.4.

Maybe it's also time to drop Symfony 4.4? since we are also supporting another LTS (5.4)

---

previous doctrine behavior versions also support dbal 3 , so it is fine, afaik

---

https://github.com/sonata-project/SonataTranslationBundle/actions/runs/4382455582/jobs/7671520120 this is what I meant, we cannot run the job with symfony 4.4

---

then you might want to allow dbal 2 and 3, for the moment.

Then we can discuss about dropping symfony 4.4. (I would wait to another round of PRs, since this migration to php8.0 is already taking some time).

wdyt @VincentLanglet ?